.github/workflows/action-build.yml

```diff
@@ -28,7 +28,7 @@ jobs:
 
       - uses: actions/setup-node@v3
```

Comment:
wonder if this version could be bumped to:

```suggestion
        uses: actions/setup-node@v6
```

Reply:
Same here opened a dedicated pr : https://github.com/release-drafter/release-drafter/pull/1468